### PR TITLE
Add navigation buttons to detail view of the metadata editor

### LINKED ISSUE #5930: Add buttons to browse the files in the detail view of the metadata editor
**Is your feature request related to a problem? Please describe.**
In Kitodo.Production 3 it is difficult to navigate in the detail view of the metadata editor - especially for users who are not used to apply short keys. 
This has been discussed in: 
- https://github.com/kitodo/kitodo-production/discussions/5881

**Describe the solution you'd like**
Two buttons should be created for the detail view to move forward and backward in the files. 
- For each klick on the "forward-button": the next file should be shown. 
- For each klick on the "backward-button": the file before should be shown.

### PR DESCRIPTION
**Pull request by thomaslow**
Fixes #5930 

Todo
- [x] Button titles with translation
- [x] Disable buttons if first or last page
- [x] Selenium test

Demo video:

https://github.com/user-attachments/assets/14f2e2a1-b4e2-4f98-bff6-6732dcb76022

### CODE REVIEW

**Comment by solth** [comment]
@thomaslow do you still plan to change something with this pull request? Or is there another reason why it isn't ready for review, yet?

**Review by solth — COMMENTED** [review]
Works and code looks good, thank you. Just a few very minor suggestions. 

I am unsure about German umlauts in the `messages_de.properties` file, though. With the correct file encoding, these are displayed correctly, but we have had several cases in the past where the encoding of these files was changed inadvertently. Since Git doesn't detect changes in the encoding, this has led to many cases where special characters wouldn't be displayed correclty anymore. 

Therefor we have tried to stick to using the codepoints instead of the special characters they encode (so `f\u00FCr` instead of `für`) to prevent this problem until we find a more reliable solution for this problem.

**Comment by thomaslow** [comment]
> we have had several cases in the past where the encoding of these files was changed inadvertently

@solth Do you mean developers have accidentally added incorrectly encoded (e.g. latin-1 encoded) umlauts to the file? Or do you mean that Kitodo.Production was installed on a system that caused the correctly encoded UTF-8 messages file to be loaded incorrectly?

In my opinion, the first case can be covered by some simple unit tests.

File: Kitodo/src/main/resources/messages/messages_de.properties [hunk 1]
```diff
@@ -334,6 +334,10 @@ dataEditor.layoutMenuSaveForTaskText=Spaltenaufteilung für Aufgabentyp speicher
 dataEditor.layoutSavedSuccessfullyTitle=Aktuelle Spaltenaufteilung erfolgreich gespeichert
 dataEditor.layoutSavedSuccessfullyDefaultText=Ihre Einstellungen wurden erfolgreich als Standard-Einstellungen gespeichert! Beim n\u00E4chsten \u00D6ffnen des Matadaten-Editors wird die Breite der drei Spalten (Strukturdaten, Metadaten und Galerie) entsprechend ihrer aktuellen Einstellung geladen.
 dataEditor.layoutSavedSuccessfullyForTaskText=Ihre Einstellungen wurden erfolgreich f\u00FCr alle zuk\u00FCnftigen Aufgaben des Typs "{0}" gespeichert! Beim n\u00E4chsten \u00D6ffnen des Matadaten-Editors f\u00FCr eine Aufgabe des gleichen Typs wird die Breite der drei Spalten (Strukturdaten, Metadaten und Galerie) entsprechend ihrer aktuellen Einstellung geladen.
+dataEditor.navigateToPreviousElementMany=mehrere Elemente zurück springen
```
[the remark this answers is not in the document]
Comment (thomaslow) [reply Kitodo/src/main/resources/messages/messages_de.properties]: Done and `weiter springen` to `vorspringen`.

File: Kitodo/src/test/java/org/kitodo/selenium/MetadataImagePreviewST.java [hunk 2]
```diff
@@ -209,6 +211,77 @@ public void viewPersistsImageChange() throws Exception {
...
+            () -> "Bild 3, Seite -".equals(
+                findElementsByCSS(GALLERY_HEADING_WRAPPER_SELECTOR).get(0).getText().strip()
+            )
+        );
+
+        // find buttons again because image preview is re-rendered
+        WebElement leftSlow = findElementsByCSS("#imagePreviewForm\\:navigateToPreviousElementOne").get(0);
+        WebElement rightSlow = findElementsByCSS("#imagePreviewForm\\:navigateToNextElementOne").get(0);
+
+        // check left buttons are enalbed and right buttons are disabled
```
[the remark this answers is not in the document]
Comment (thomaslow) [reply Kitodo/src/test/java/org/kitodo/selenium/MetadataImagePreviewST.java]: Done

File: Kitodo/src/main/java/org/kitodo/production/forms/dataeditor/GalleryPanel.java [hunk 3]
```diff
@@ -1048,4 +1048,60 @@ public boolean hasMediaViewMimeTypePrefix(String mimeTypePrefix) {
...
+        PhysicalDivision firstPhysicalDivision = medias.get(medias.size() - 1).getView().getPhysicalDivision();
+        if (Objects.isNull(firstPhysicalDivision)) {
+            return false;
+        }
+
+        if (firstPhysicalDivision.equals(lastSelection.getKey())) {
+            return true;
+        }
+
+        return false;
```
[the remark this answers is not in the document]
Comment (thomaslow) [reply Kitodo/src/main/java/org/kitodo/production/forms/dataeditor/GalleryPanel.java]: Done

**Review by solth — APPROVED** [review]
Thank you for the adjustments. 

> @solth Do you mean developers have accidentally added incorrectly encoded (e.g. latin-1 encoded) umlauts to the file? Or do you mean that Kitodo.Production was installed on a system that caused the correctly encoded UTF-8 messages file to be loaded incorrectly?
> 
> In my opinion, the first case can be covered by some simple unit tests.

Yes, I was talking about the first case and was hoping we can ensure correct encodings (in the repository) and character displays using automated tests. Once these are in place, we can switch to including special characters like umlauts in the message files.

**Comment by solth** [comment]
@thomaslow would you mind opening a corresponding pull request against the `3.8.x` branch of Kitodo.Production? Since the `master` branch hasn't deviated very much from it, yet, there hopefully shouldn't be any conflicts or problems. That way the final `3.8.0` release would contain all new features from the metadata bundle (apart from the structure tree multi select, which will come at a later point).

**Pull request merged**